src/screens/bottom-sheet/accounts-bottom-sheet/accounts-bottom-sheet.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { View } from 'react-native';
+import { withTheme, IThemeProps } from '../../../core/theme/with-theme';
+import stylesProvider from './styles';
+import { smartConnect } from '../../../core/utils/smart-connect';
+import BottomSheet from 'reanimated-bottom-sheet';
+import { AccountsScreen } from '../../accounts/accounts';
+import { Icon } from '../../../components/icon';
+
+const BOTTOM_SHEET_HEIGHT = 600;
+
+interface IProps {
+    onClose: () => void;
+}
+
+export class AccountsBottomSheetComponent extends React.Component<
+    IProps & IThemeProps<ReturnType<typeof stylesProvider>>
+> {
+    public bottomSheet: any;
+
+    constructor(props: IProps & IThemeProps<ReturnType<typeof stylesProvider>>) {
+        super(props);
+        this.bottomSheet = React.createRef();
+    }
+
+    public componentDidMount() {
+        this.bottomSheet.current.snapTo(BOTTOM_SHEET_HEIGHT);
+    }
+
+    public renderBottomSheetContent = () => (
+        <View style={{ height: BOTTOM_SHEET_HEIGHT }}>
+            <AccountsScreen />
```

Comment:
this is not a screen anymore, if the component is not very complex, move the code here, otherwise move the component here